lib/emscripten/src/lib.rs

```diff
@@ -104,1116 +112,1235 @@ macro_rules! mock_external {
     }};
 }
 
+struct EmscriptenGlobals {
+    pub data: Vec<(String, LocalGlobal, GlobalDesc)>,
+}
+
+impl EmscriptenGlobals {
```

Comment:
I don't think this abstraction is really necessary.

Reply:
Responded below